dbt/seeds/ccao/schema.yml

```diff
@@ -0,0 +1,9 @@
+seeds:
```

Comment:
**suggestion:** Let's import the rest of the column-level descriptions from https://github.com/ccao-data/ccao/blob/969dae702ed420ba9f9d252e5a0459c63b991e80/R/data.R#L75